tools/cleanup/README.md

```diff
@@ -0,0 +1,31 @@
+# Resources Clean Up Tool
+This tool is designed to clean up staled resources from gcr, for now it only deletes old images created during testing.
+
+## Basic Usage
+Directly invoke [cleanup.sh](cleanup.sh) script with certain flags. There is no-op if invoking or sourcing this script without argumements.
+
+### Clean up old images from multiple projects
+Projects to be cleaned up are expected to be defined in a `resources.yaml` file. To remove old images from them, call [cleanup.sh](cleanup.sh) with action "delete-old-gcr-images" and following flags:
+- "--project-resource-yaml" as path of `resources.yaml` file - Mandatory
+- "--re-project-name" for regex matching projects names - Optional, default `knative-boskos-[a-zA-Z0-9]+`
+- "--days-to-keep" - Optional, default `365`
+
+Example:
+
+```./cleanup.sh "delete-old-gcr-images" --project-resource-yaml "ci/prow/boskos/resources.yaml" --days-to-keep 90```
+
+### Clean up old images from specified project
```

Comment:
As I mentioned before "project" makes me think of a GCP project. Please use "gcr" instead, here and throughout this doc.